src/deadline/job_attachments/asset_sync.py

```diff
@@ -166,7 +166,7 @@ def get_local_destination(
                 f"No path mapping rule found for the source path {manifest_properties.rootPath}"
             )
 
-    def aggregate_asset_root_manifests(
+    def _aggregate_asset_root_manifests(
```

Comment:
This is changing the public interface of the library. If you do this, your commit and CR need to be `feat!` instead of `feat`, and you need to describe what the backwards incompatible change is.

Reply:
Makes sense, will update to reflect the breaking change.